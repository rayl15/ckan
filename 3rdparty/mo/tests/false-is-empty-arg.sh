#!/usr/bin/env bash

cd "${0%/*}"
USER=j.doe ADMIN=false ../mo --false false-is-empty-arg.template
