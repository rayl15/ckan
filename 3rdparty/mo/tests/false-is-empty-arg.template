The user {{USER}} exists.
{{#ADMIN}}
WRONG - should not be an admin.
{{/ADMIN}}
