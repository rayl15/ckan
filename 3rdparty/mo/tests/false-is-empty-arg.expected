The user j.doe exists.
